specifications/api/swagger.yaml

```diff
@@ -1474,6 +1525,37 @@ components:
           passphrase: *lenientPassphrase
           metadata: *transactionMetadata
 
+    ApiScript: &ApiScript
+      type: object
+      required:
+        - script
+      properties:
+        script: *script
+
+    ApiPubKey: &ApiPubKey
+      type: object
+      required:
+        - pub_key
+      properties:
+        pub_key: *credentialPubKey
```

Comment:
I think these could both be removed and embedded directly into `CredentialValue`

Reply:
I wanted them to be exposed to have a way in TypeSpec to check

Reply:
OK fair enough